packages/fx-core/src/plugins/resource/apim/constants.ts

```diff
@@ -266,6 +284,7 @@ export const PluginLifeCycleToProgressStep: { [key in PluginLifeCycle]: Progress
   [PluginLifeCycle.GetQuestions]: ProgressStep.None,
   [PluginLifeCycle.Scaffold]: ProgressStep.Scaffold,
   [PluginLifeCycle.Provision]: ProgressStep.Provision,
+  [PluginLifeCycle.GenerateArmTemplates]: ProgressStep.GenerateArmTemplates,
```

Comment:
You can set `[PluginLifeCycle.GenerateArmTemplates]: ProgressStep.None` here and remove `[ProgressStep.GenerateArmTemplates]: {},`